packages/react/src/components/Layer/Layer.base.tsx

```diff
@@ -24,6 +26,14 @@ import type { ILayerProps, ILayerStyleProps, ILayerStyles } from './Layer.types'
 
 const getClassNames = classNamesFunction<ILayerStyleProps, ILayerStyles>();
 
+const getFocusVisibility = (providerRef?: React.RefObject<HTMLElement>) => {
+  if (providerRef?.current) {
+    return providerRef.current.classList.contains(IsFocusVisibleClassName);
+  }
+
+  return false;
+};
```

Comment:
There is no focus visible state saved anywhere in Fluent -- it's done purely through setting the className based on user events. The only way to check the current state is to check for the presence of the className.